spec/fast_inserter_base_spec.rb

```diff
@@ -85,7 +121,7 @@
 
       # Make sure that each is created, one per user, and each has timestamps set properly
       expect(event.attendees.count).to eq 4
-      expect(event.attendees.pluck(:user_id)).to match_array user_ids
+      expect(event.attendees.pluck(:user_id)).to match_array user_ids.uniq
```

Comment:
Previously, user_ids was mutated by the `fast_insert` call; this no longer happens so we need to call uniq here.